core/output/slack.js

```diff
@@ -206,6 +205,68 @@ init = function(token, callback) {
 	});
 },
 
+startPingPongTimer = function(teamId) {
+	if (!Date.now) {
+    	Date.now = function() {
+			return new Date().getTime();
+		}
+	}
+
+	var team = exports.config.slack_teams[teamId],
+		messageId = ++team.ping_id;
+
+	if (!messageId) {
+		messageId = 0;
+		team.ping_id = messageId;
+	}
+
+	team.timeSincelastMessageRecieved = Date.now();
+	setTimeout(() => {
+		if (Date.now() - team.timeSincelastMessageRecieved >= 60000 && messageId === team.ping_id) {
+			console.debug('sending ping');
+			sendPing(teamId);
+		}
+	}, 60000);
+},
+
+sendPing = function(teamId) {
+	var socket = sockets[teamId],
+		slackTeams = exports.config.slack_teams,
+		team = slackTeams[teamId],
+		pongId = ++team.pong_id;
+
+	if (!pongId) {
+		pongId = 0;
+		team.pong_id = pongId;
+	}
+
+	if (socket != null) {
+		var body = {
+		'id': team.pong_id,
```

Comment:
`var body = {` and `'id': team.pong_id` are on the same indent level, when the second should be one further in. If it is just showing up inconsistently here it will be atom switching between tabs and spaces.